Tests/XcodeprojTests/FunctionalTests.swift

```diff
@@ -147,8 +147,10 @@ func XCTAssertXcodeBuild(project: AbsolutePath, file: StaticString = #file, line
         if env["TOOLCHAINS"] == nil {
             env["TOOLCHAINS"] = "default"
         }
+        let schemeName = String(project.basename.dropLast(10)) + "-Package"
+        let buildDir = project.parentDirectory.appending(component: "build").asString
         try Process.checkNonZeroExit(
-            args: "xcodebuild", "-project", project.asString, "-alltargets", environment: env)
+            args: "xcodebuild", "-project", project.asString, "-scheme", schemeName, "-derivedDataPath", buildDir, environment: env)
```

Comment:
Can we have a test that checks that the dummy target builds, by passing `-target`? That way we can validate it is configured properly.

Also, should we put *all* manifests in it, as opposed to just the one from the primary package?